h2o-py/h2o/h2o.py

```diff
@@ -400,6 +400,7 @@ def import_file(path=None, destination_frame=None, parse=True, header=0, sep=Non
     :param pattern: Character string containing a regular expression to match file(s) in the folder if `path` is a
         directory.
     :param skipped_columns: an integer list of column indices to skip and not parsed into the final frame from the import file.
+    :param non_data_line_markers: If a line in imported file starts with any character in given string it will be imported. Empty string means all lines are imported, None means that default behaviour for given format will be used
```

Comment:
I think this is not correct, correct is:

If a line in imported file starts with any character in given string it will NOT be imported.

Reply:
Sorry, I copy&pasted your proposal without reading it.